mmcv/cnn/bricks/conv.py

```diff
@@ -36,11 +35,11 @@ def build_conv_layer(cfg: Optional[Dict], *args, **kwargs) -> nn.Module:
         cfg_ = cfg.copy()
 
     layer_type = cfg_.pop('type')
-    if layer_type not in CONV_LAYERS:
-        raise KeyError(f'Unrecognized layer type {layer_type}')
-    else:
-        conv_layer = CONV_LAYERS.get(layer_type)
-
+    with MODELS.switch_scope_and_registry(None) as TARGET_MODELS:
+        conv_layer = TARGET_MODELS.get(layer_type)
```

Comment:
```suggestion
    with MODELS.switch_scope_and_registry(None) as registry:
        conv_layer = registry.get(layer_type)
```
Also add some commnets